### PR TITLE
update edu benefits api doc

### LINKED ISSUE #89: edu benefits api doc

### PR DESCRIPTION
**Pull request by lihanli**
closes #89

### CODE REVIEW

File: docs/api/edu-benefits.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  "data": {
+    "id": "18",
+    "type": "education_benefits_claims",
+    "attributes": {
+      // the form that was submitted
+      "form": "{\"preferredContactMethod\":\"mail\"}",
+      // when it was first submitted
+      "submittedAt": "2016-09-09T23:48:07.766Z",
+      // when it was sent to the VA SFTP for processing
+      "processedAt": null
```
Comment (markolson) [inline docs/api/edu-benefits.md]: I think `processedAt` will always be null -- should we be including it?

Comment (lihanli) [inline docs/api/edu-benefits.md]: i'll make another PR for removing it

**Comment by mphprogrammer** [comment]
My apologies - I accidentally merged master into this branch after seeing that this branch was out of date with master, and thinking it was my own PR.

**Comment by lihanli** [comment]
@markolson ok, i updated the doc to get rid of processedAt

**Pull request merged**